tests/scripts/thread-cert/Cert_9_2_04_ActiveDataset.py

```diff
@@ -182,6 +188,96 @@ def test(self):
         for ipaddr in ipaddrs:
             self.assertTrue(self.nodes[LEADER].ping(ipaddr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        COMMISSIONER = pv.vars['COMMISSIONER']
+
+        # Step 1: Ensure the topology is formed correctly
+        pkts.filter_wpan_src64(LEADER).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+
+        # Step 2: Commissioner sends MGMT_ACTIVE_SET.req to Leader RLOC
+        pkts.filter_wpan_src64(COMMISSIONER).filter_coap_request(MGMT_ACTIVE_SET_URI).must_next().must_verify(
+            lambda p: bytes.fromhex(str(format(101, 'x'))) in p.coap.payload and 'GRL'.encode() in p.coap.payload)
+
+        # Step 3: Leader MUST send MGMT_ACTIVE_SET.rsp to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_ACTIVE_SET_URI).must_next()
+
+        # Step 4: Commissioner sends MGMT_ACTIVE_GET.req to Leader
+        pkts.filter_wpan_src64(COMMISSIONER).filter_coap_request(MGMT_ACTIVE_GET_URI).must_next()
+
+        # Step 5: The Leader MUST send MGMT_ACTIVE_GET.rsp to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_ACTIVE_GET_URI).must_next().must_verify(
+            lambda p: bytes.fromhex(str(format(101, 'x'))) in p.coap.payload and 'GRL'.encode() in p.coap.payload)
+
+        # Step 6: Commissioner sends MGMT_ACTIVE_SET.req to Leader RLOC or Anycast Locator
+        pkts.filter_wpan_src64(
```

Comment:
Validate destination is Leader RLOC or Anycast Locator? (many more instances below)